tests/integration/test_hyper_transformer.py

```diff
@@ -207,6 +207,9 @@ def test_hypertransformer_default_inputs():
     assert isinstance(ht._transformers_tree['names']['transformer'], FrequencyEncoder)
     assert ht._transformers_tree['names']['outputs'] == ['names.value']
 
+    get_default_transformers.cache_clear()
```

Comment:
We need these lines because `get_default_transformer(s)` caches the output of the function, but since this test patches the method, it ends up caching the patched, incorrect value, which makes any tests that run after it fail.

Deleting the cache fixes the problem.